cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,48 @@
+/// <reference types='cypress' />
+
 describe('', () => {
-  before(() => {
+  let user;
 
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+      cy.login(user.email, user.username, user.password);
+      cy.visit('/');
+    });
   });
 
-  it('', () => {
-
+  it('should have the ability to create an article', () => {
+    let article;
+    let usernameToLowerCase = user.username.toLowerCase();
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+      cy.createArticle(article.title, article.about, article.body);
```

Comment:
```suggestion
```
You had to check article creation via ui. Need to visit new article page and fill the fields